.github/workflows/bump.yml

```diff
@@ -7,7 +7,7 @@ on:
     paths:
       - 'oas_docs/kibana.serverless.yaml'
 
-  pull_request:
+  pull_request_target:
```

Comment:
IMO, this is a no-go when using GitHub secrets, see https://nathandavison.com/blog/github-actions-and-the-threat-of-malicious-pull-requests

Reply:
For future readers, I've answered the concerns in this reply: https://github.com/elastic/kibana/pull/186909#issuecomment-2215371544